student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(_classroom, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] I would recommend you have `classroom` (without the underscore) to match the instantiation you are creating.

Reply:
Thank you @TedLivist  am going to work on the requested changes